web/modules/custom/dbcdk_community/src/Content/FlaggableContent.php

```diff
@@ -0,0 +1,121 @@
+<?php
+
+/**
+ * @file
+ * FlaggableContent class definition.
+ */
+
+namespace Drupal\dbcdk_community\Content;
+
+
+use DBCDK\CommunityServices\Model\Flag;
+
+/**
+ * A pience of community content with 0 or more flags attached.
+ *
+ * Users can flag content to bring it to attention of the moderators
+ * of the site.
+ *
+ * @package Drupal\dbcdk_community\Content
+ */
+class FlaggableContent {
+
+  protected $object;
+
+  protected $flags = [];
+
+  /**
+   * FlaggableContent constructor.
+   *
+   * @param object $object
+   *   The flaggable object.
```

Comment:
A description of why we do not use type-cast for this "object" - I know it's because it can be multiple types of content but it is accumulated knowledge not documentation :)